source/options.tsx

```diff
@@ -173,10 +174,36 @@ async function highlightNewFeatures(): Promise<void> {
 	void browser.storage.local.set({featuresAlreadySeen});
 }
 
+async function getFeaturesDisabledViaHotfix(): Promise<HTMLElement[]> {
+	const hotfixes = await cache.get<string[][]>('hotfixes');
+	if (!hotfixes) {
+		return [];
+	}
+
+	const {version} = browser.runtime.getManifest();
+	const featureNames: Set<string> = new Set(features.map(({id}) => id));
```

Comment:
Used a `Set` here only because the linter wanted one.

Reply:
Alternatively you can `features.find({id} => id === featureName)` as a condition directly